scripts/drone/rgm.star

```diff
@@ -73,6 +73,25 @@ tag_trigger = {
     },
 }
 
+nightly_trigger = {
+    "event": {
+        "include": [
```

Comment:
Do you have to use `include` here or could you just do something like `"event": ["promote", "cron"],`? Not sure what the difference here is 🙂

Reply:
Tried it using `"event": ["promote", "cron"]` didn't seem to work, maybe I can't have 2 events on the same trigger? It works now. But you're correct, theres no difference https://docs.drone.io/pipeline/triggers/